app/controllers/page_views_controller.rb

```diff
@@ -46,7 +46,7 @@ def page_view_params
   def update_organic_page_views
     return if Rails.env.production? && rand(100) != 1 # We need to do this operation only once in a while.
 
-    page_views_from_google_com = @article.page_views.where(referrer: "https://www.google.com/")
+    page_views_from_google_com = @article.page_views.where(referrer: "https://www.google.com")
```

Comment:
@vaidehijoshi this is going to cause a problem because we are querying for existing articles that match this exact referrer so we can't change this.